apps/src/labs/projects/ProjectManager.ts

```diff
@@ -208,15 +227,23 @@ export default class ProjectManager {
     return true;
   }
 
-  private enqueueSave() {
-    this.saveQueued = true;
-
-    this.currentTimeoutId = window.setTimeout(
-      () => {
-        this.saveHelper();
-      },
-      this.nextSaveTime ? this.nextSaveTime - Date.now() : this.saveInterval
-    );
+  private enqueueSaveOrSave(forceSave: boolean) {
```

Comment:
This is an intriguing function; it might be worth a comment explaining what it does, and what `forceSave` does?

Reply:
done!